server/controllers/availability.js

```diff
@@ -0,0 +1,168 @@
+const { Schedule, Availability } = require("../models/availability");
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+const match = require("nodemon/lib/monitor/match");
+const { getAvailabilityData } = require("../utils/availabilityMockupData");
+
+// @route POST /availability/create
+// @desc Registers pet sitter availability
+// @access Private
+exports.registerAvailability = asyncHandler(async (req, res, next) => {
+  const {
+    name,
+    isActive,
+    monday,
+    tuesday,
+    wednesday,
+    thursday,
+    friday,
+    saturday,
+    sunday,
```

Comment:
If days is an object with days as it's elements. I think spreading days object in  `res.body`  will be fine. just my opinion. I will ask for clarification.

Reply:
If we just package all the days in a `days` object in the request payload it will be a lot cleaner and reduce our controller code more. It's just keeping this more clean.